doc/LLC/LLC_Quickstart_Tutorial.md

```diff
@@ -0,0 +1,181 @@
+# LLC Quickstart Tutorial
```

Comment:
Let's use this document to try out the new "data plane code generation" name, and see how it works in this context.

Reply:
Renamed LLC to "Dataplane Code Gen" in the doc. I'll update the template project links after we merge https://github.com/Azure/azure-sdk-for-net/pull/25390 PR.